configure.ng

```diff
@@ -723,7 +721,8 @@ AC_DEFINE_UNQUOTED(HOST_OS, "$host_os" )
 
 # Add additional CFLAGS, LDFLAGS and LIBS which were specified on the command
 # line or by some tests from above, but after running this script. Useful for
-# adding "-Werror", for example:
+# adding "-march=native", for example:
+CFLAGS="$CFLAGS -pipe -Werror -Wall -Wextra -pedantic"
```

Comment:
> not all compilers support all of those flags (ngIRCd even supports pre-ANSI C compilers).

I know it supports pre-ANSI C compilers, but realistically, should the tiny minority of supported systems prevent enabling better warnings to catch more human errors (and possible security issues) for releases 28+? If a really old system really only had older compilers that didn't support these (relatively widespread) warning flags, then they can continue using release 27 since anyway, their system probably does not support newer dependencies like OpenSSL?